app/javascript/dashboard/components/widgets/conversation/ConversationHeader.vue

```diff
@@ -1,53 +1,55 @@
 <template>
   <div class="conv-header">
-    <div class="user">
-      <back-button v-if="showBackButton" :back-url="backButtonUrl" />
-      <Thumbnail
-        :src="currentContact.thumbnail"
-        size="40px"
-        :badge="inboxBadge"
-        :username="currentContact.name"
-        :status="currentContact.availability_status"
-      />
-      <div class="user--profile__meta">
-        <h3 class="user--name text-truncate">
-          <span class="margin-right-smaller">{{ currentContact.name }}</span>
-          <fluent-icon
-            v-if="!isHMACVerified"
-            v-tooltip="$t('CONVERSATION.UNVERIFIED_SESSION')"
-            size="14"
-            class="hmac-warning__icon"
-            icon="warning"
-          />
-        </h3>
-        <div class="conversation--header--actions">
-          <inbox-name
-            v-if="hasMultipleInboxes"
-            :inbox="inbox"
-            class="margin-right-small"
-          />
-          <span
-            v-if="isSnoozed"
-            class="snoozed--display-text margin-right-small"
-          >
-            {{ snoozedDisplayText }}
-          </span>
-          <woot-button
-            class="user--profile__button margin-right-small"
-            size="small"
-            variant="link"
-            @click="$emit('contact-panel-toggle')"
-          >
-            {{ contactPanelToggleText }}
-          </woot-button>
+    <div class="conversation-header--details">
+      <div class="user">
+        <back-button v-if="showBackButton" :back-url="backButtonUrl" />
+        <Thumbnail
+          :src="currentContact.thumbnail"
+          size="40px"
```

Comment:
```suggestion
```

`Thumbnail` already has a default size of `40px`. 

On another note, not actionable for this PR. 
Passing value for CSS styles as a prop is generally considered an anti-pattern, given one cannot reliably verify the value, WDYT?

CC: @fayazara